luminoso_api/json_stream.py

```diff
@@ -237,10 +273,9 @@ def _read_csv(reader, header, encode_fn):
         if row_dict.get('title') == '':
             del row_dict['title']
         if 'date' in row_dict:
+            # We handle dates further in stream_json_lines
```

Comment:
...so is this comment still true?  Are the dates further handled somewhere else?